addons/account_edi/models/account_move.py

```diff
@@ -36,6 +36,8 @@ class AccountMove(models.Model):
         compute='_compute_edi_show_cancel_button')
     edi_show_abandon_cancel_button = fields.Boolean(
         compute='_compute_edi_show_abandon_cancel_button')
+    edi_show_force_cancel_button = fields.Boolean("Force Cancel",
```

Comment:
```suggestion
    edi_show_force_cancel_button = fields.Boolean(
```
nitpicking: the other `edi_show_*_button` fields don't have a string, so maybe this one doesn't need one either